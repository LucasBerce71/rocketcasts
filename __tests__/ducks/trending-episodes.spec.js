import ActionCreators, { reducer, INITIAL_STATE } from 'store/ducks/trending-episodes';

test('request', () => {
  const state = reducer(INITIAL_STATE, ActionCreators.trendingRequest());

  expect(state).toMatchSnapshot();
});

test('success', () => {
  const state = reducer(INITIAL_STATE, ActionCreators.trendingSuccess(['result']));

  expect(state).toMatchSnapshot();
});

test('failure', () => {
  const state = reducer(INITIAL_STATE, ActionCreators.trendingFailure());

  expect(state).toMatchSnapshot();
});
