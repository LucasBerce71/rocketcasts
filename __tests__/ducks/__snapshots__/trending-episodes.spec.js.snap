// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`failure 1`] = `
Object {
  "data": Array [],
  "error": true,
  "loading": false,
}
`;

exports[`request 1`] = `
Object {
  "data": Array [],
  "error": null,
  "loading": true,
}
`;

exports[`success 1`] = `
Object {
  "data": Array [
    "result",
  ],
  "error": null,
  "loading": false,
}
`;
